addons/project/report/project_report_views.xml

```diff
@@ -60,7 +60,7 @@
                     <separator/>
                     <filter string="Starred" name="starred" domain="[('priority', 'in', [0, 1])]"/>
                     <separator/>
-                    <filter string="Tasks Late" name="late" domain="[('date_deadline', '&lt;', context_today().strftime('%Y-%m-%d'))]"/>
+                    <filter string="Tasks Late" name="late" domain="[('date_deadline', '&lt;', context_today().strftime('%Y-%m-%d')), ('task_id.is_closed', '=', False)]"/>
```

Comment:
If I am not mistaken the field is in the report and stored from saas-15.3+
Could you have a look?
Thanks!

Reply:
Yes it is!
https://github.com/odoo/odoo/blob/saas-15.3/addons/project/report/project_report.py#L48